CC* variables not defined

Reproduced via setting AT86RF2XX_BOARD=SNR ...


The output:
```
../../os/sys/etimer.h:252:1: warning: type defaults to 'int' in declaration of 'PROCESS_NAME'
../../os/sys/etimer.h:252:1: warning: parameter names (without types) in function declaration
cc1: warning: unrecognized command line option "-Wno-unused-const-variable"
  CC        ../../arch/platform/vesna/dev/watchdog.c
  CC        ../../arch/platform/vesna/dev/node-id.c
  CC        ../../arch/platform/vesna/dev/slip-arch.c
  CC        ../../arch/platform/vesna/dev/at86rf2xx/rf2xx.c
  CC        ../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c
../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c: In function 'configure_cc1101':
../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c:599:9: error: 'CC1101_CSN_PIN' undeclared (first use in this function)
         CC1101_CSN_PIN,
         ^
../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c:599:9: note: each undeclared identifier is reported only once for each function it appears in
../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c:600:9: error: 'CC1101_CSN_PORT' undeclared (first use in this function)
         CC1101_CSN_PORT,
         ^
../../arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.c: At top level:
cc1: warning: unrecognized command line option "-Wno-unused-const-variable"
../../Makefile.include:353: recipe for target 'build/vesna/obj/rf2xx_hal.o' failed
make: *** [build/vesna/obj/rf2xx_hal.o] Error 1
```

--- a/arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.h
+++ b/arch/platform/vesna/dev/at86rf2xx/rf2xx_hal.h
@@ -123,15 +123,16 @@
 	#define EXTI_IRQ_PIN		(GPIO_PinSource3)
 	#define EXTI_IRQ_CHANNEL	(EXTI3_IRQn)
 
-	// Chip select pins for CC101 radio
-	#define CC1101_CSN_PIN      (GPIO_Pin_9)
-	#define CC1101_CSN_PORT     (GPIOB)
-
-
 #else
 	#warning "No predefined board was selected!"
 	#error "No pins have been defined for AT86RF2xx radio!"
 #endif
+
+
+// TODO: Separate CC and Atmel radio configs
+// Chip select pins for CC101 radio
+#define CC1101_CSN_PIN      (GPIO_Pin_9)
+#define CC1101_CSN_PORT     (GPIOB)
 
 typedef union {
 	struct {

